src/main/java/org/codinjutsu/tools/jenkins/view/JenkinsTreeNode.java

```diff
@@ -50,6 +52,22 @@ default boolean canNavigateToSource() {
 
     void render(JenkinsTreeNodeVisitor treeNodeRenderer);
 
+    @Data
+    class BuildParameterNode implements JenkinsTreeNode {
+        private final BuildParameter buildParameter;
+
+        @NotNull
+        @Override
+        public String getUrl() {
+            return "";
+        }
```

Comment:
Should this return something else than an empty string? (may be the URL to the job?
maxe be `getName` should overridden to return the parameter name.

Reply:
`getUrl` return build url will be good